src/Sylius/Behat/Context/Ui/Admin/ManagingPromotionsContext.php

```
@@ -778,4 +778,11 @@ protected function resolveCurrentPage(): SyliusPageInterface
     {
         return $this->currentPageResolver->getCurrentPageWithForm([$this->createPage, $this->updatePage]);
     }
+    #[When('/^I add the "Cart quantity" rule with minimum (\d+) items for ("(?:[^"]+)") channel$/')]
+    public function iAddTheCartQuantityRuleWithMinimumItemsForChannel(int $quantity, ChannelInterface $channel): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$channel'. (undefined)

(UnusedFormalParameter)